audit_report/3.1.0/changes/topics/additions.yml

```diff
@@ -105,14 +105,19 @@ patches:
 # Add vpternlogd based Serpent sboxes  (@randombit)
 - pr: 3536  # https://github.com/randombit/botan/pull/3536
   merge_commit: 8b3dff8896c9f157c86c7469702acd1519e7752a
-  classification: unspecified
+  classification: out of scope
```

Comment:
I think it would be nice to give a short reason why something is out of scope (like here: "Serpent prohibited by BSI policy".) Can you come up with an easy solution to include this info in the future? Maybe just in the comment?

Reply:
IMO "out of scope" means by default, that it does not touch any modules that is listed in our "modules to review" list. I think it is not sensible to write a comment for each out-of-scope patch with a similar reasoning. But I see your point. I would suggest to explain this deeper in the introduction chapters.

Reply:
If you want, we can discuss the "out of scope" reasoning within a separate ticket or in our next Jour Fixe. If we agree on a change we can create a new PR to address the other "out of scope"s as well. Then, we can merge this PR now.